infrastructure-elemental/v0.0.0/cluster-template-rke2.yaml

```diff
@@ -80,42 +77,11 @@ spec:
   nodeDrainTimeout: 2m
   registrationMethod: "address"
   registrationAddress: "${CONTROL_PLANE_ENDPOINT_HOST}"
+  rolloutStrategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxSurge: 1
   files:
-  - path: /var/lib/rancher/rke2/server/manifests/kube-vip-rbac.yaml
```

Comment:
RBAC is only needed when deploying kube-vip as DaemonSet, but in this case we are deploying it as a static Pod, so this is not needed.